### PR TITLE
update eslint, allow tabs

### PR DESCRIPTION
**Pull request by rryoung98**
Updated eslint, relaxed rules for indentation, tabs, and react. We can enforce them later

### CODE REVIEW

File: frontend/.eslintrc.json [hunk 1]
```diff
@@ -18,6 +18,11 @@
 	},
 	"plugins": ["react", "react-hooks"],
 	"rules": {
+		"no-tabs": "off",
+		"react/jsx-indent": "off",
+		"react/jsx-indent-props": "off",
+		"react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+		"comma-dangle": ["error", "never"]
 		//add customize rules here as per your project's needs
```
Comment (smburdick) [inline frontend/.eslintrc.json:26]: This comment is not needed.

**Review by smburdick — APPROVED** [review]
LGTM. I am personally ok with only using .jsx for React files, but we didn't start with that for this app (I did rename App.js to App.jsx in #83 , though)

File: frontend/.eslintrc.json (hunk 1 above)
Comment (rryoung98) [reply frontend/.eslintrc.json]: Changes made

**Comment by rryoung98** [comment]
Yea, we can switch .js / .jsx later if we want, just don't want our eslint to bog us down 😄

**Pull request merged**